mitodl/bootcamp-ecommerce
# Rename a couple UWSGI env vars, remove redundant if-env blocks

#### What are the relevant tickets?
Related to https://github.com/mitodl/micromasters/issues/4597

#### What's this PR do?
 - Renames `UWSGI_THREAD_COUNT` to `UWSGI_THREADS`
 - Removes redundant if-env blocks

#### How should this be manually tested?
If you have either of the above UWSGI_ environment variables, rename them. Then load the site and make sure it functions.
